### ISSUE TITLE
Can issues be imported?

### ISSUE THREAD

**Issue by iamso** [sgreben/jira-project-export#2]
Hi

Can issues be imported into a different Jira account/instance? 
If yes, how?

Thanks

**Comment by sgreben** [comment]
Hi, I think one could probably use the [bulk issue creation endpoint](https://docs.atlassian.com/software/jira/docs/api/REST/7.6.1/#api/2/issue-createIssues):

> POST /rest/api/2/issue/bulk
> Creates issues or sub-tasks from a JSON representation.

You would probably need to process the export JSON a bit to match the create schema, but it looks doable.

If you end up figuring this out, I'd greatly appreciate a PR adding this (or a link to a separate repo with the import tool). :)

**Comment by iamso** [comment]
@sgreben thank you. I will have a look at it.

**Comment by jhult** [comment]
@iamso and @sgreben any luck with this?

**Comment by sgreben** [comment]
@jhult as far as I know, no

**Comment by iamso** [comment]
@jhult no, haven't had the need to check any further, since our client only needed it for documentation purposes. Also there might have been an issue with mismatch of task IDs that are mentioned in texts and comments when you import it into another system, but not sure about that.
Anyway, I wrote a small tool to export the jira tasks and create an html file for a nicer presentation. You can check it here if you're interested https://github.com/iamso/jira-story-export.
